core/domain/classifier_services.py

```diff
@@ -359,9 +458,8 @@ def delete_classifier_training_job(job_id):
         classifier_training_job_model.delete()
 
 
-def get_classifier_from_exploration_attributes(exp_id, exp_version,
-                                               state_name):
-    """Gets the classifier object from the exploration attributes.
+def get_classifier_training_job(exp_id, exp_version, state_name):
```

Comment:
I think you'll need a get_multi version of this (corresponding to multiple states) since in general you'll need to get multiple classifiers at once for a given exploration.